frontend/src/pages/instance/components/ImportInstanceComponents.vue

```diff
@@ -99,15 +99,9 @@ export default {
             const reader = new FileReader()
             reader.onload = (evt) => {
                 try {
-                    const creds = JSON.parse(reader.result)
-                    if (creds.$) {
-                        // Good Start
-                        this.parts.credentials = reader.result
-                        this.errors.creds = ''
-                    } else {
-                        this.parts.credentials = undefined
-                        this.errors.creds = 'Does not look like a creds file'
-                    }
+                    JSON.parse(reader.result)
```

Comment:
This parse seems to be dangling. Is it just to catch it _not_ being JSON?

Reply:
Correct. Previously the code was parsing and looking for the `$` property. That is a sensible check for it being an encrypted creds file, however users may have unencrypted files which we should allow. So I reduced it to just check its valid JSON.